lib/storage/storage.dart

```diff
@@ -0,0 +1,15 @@
+import 'dart:async';
+import 'package:floor/floor.dart';
+import 'package:flutter_demo/storage/dao/exercise_dao.dart';
+import '../models/exercise.dart';
+
+
+import 'package:sqflite/sqflite.dart' as sqflite;
+
+
+part 'storage.g.dart'; // the generated code will be there
```

Comment:
U mnie, i z tego co widzę po forach to popularny problem, trzeba było puścić komendę 'flutter pub run build_runner build --delete-conflicting-outputs' żeby ta linijka nie generowała błędu